maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -191,8 +191,10 @@ class IOSDriver(
     }
 
     override fun contentDescriptor(): TreeNode {
-        logger.info("Getting view hierarchy for $appId")
-        val result = iosDevice.contentDescriptor(appId)
+        val getRunningAppId = GetRunningAppIdResolver().getRunningAppId()
+        logger.info("Getting view hierarchy for $getRunningAppId")
+
+        val result = iosDevice.contentDescriptor(appId ?: getRunningAppId)
```

Comment:
1. Let's change the var name to `runningAppId`
2. Are you keeping the value returned from `getRunningAppId` nullable? If yes then let's first depend on app id from the API response and then `appId` here which is initialized by what app is launched. (see the on success in launchApp)

Reply:
@amanjeetsingh150 it is not nullable, because we use default value com.apple.springboard if nothing is returned